README.md

```diff
@@ -64,3 +64,19 @@ To build the code with debug flags turned on (i.e., optimization turned off, deb
 This could be done in a separate directory to retain both versions of the build.
 
 See the SLiM manual for more information about building and installing, including instructions on building SLiMgui under Xcode.  The manual and other SLiM stuff can be found at [http://messerlab.org/slim/](http://messerlab.org/slim/).
+
+Instructions for parallel implementation
+----------------------------------
+-Switch to openmp branch
+```
+cd icc_build
```

Comment:
Why are these build instructions different from the build instructions given in CMakeLists.txt?

Reply:
Since I initially wanted to have tbb stuff, I had a whole set of initializations that I wrote in a build file. Now that I removed tbb, the instructions are the same as its serial counterpart with the only  change being, -D Parallel=ON